src/org/ods/orchestration/util/Project.groovy

```diff
@@ -406,6 +397,21 @@ class Project {
         return this
     }
 
+    def generateWIPIssuesMessage() {
```

Comment:
Maybe we can move the responsability of generate this message to another class instead Project class. Think that project class is too big.

Reply:
Moved to another class, and moved the tests for that specific functionallity to a new Spec for that class